src/testing.ts

```diff
@@ -90,36 +90,44 @@ export const credentials = {
 export const loggerMock = (): ActionLogger => ({
   debug: (spyOn(console, "debug") as unknown) as ActionLoggerFunction,
   info: (spyOn(console, "info") as unknown) as ActionLoggerFunction,
-  log: (spyOn(console, "log") as unknown) as ActionLoggerFunction,
```

Comment:
We'll want to add this one back in.

Reply:
Fixed: https://github.com/prismatic-io/spectral/compare/8cf1d2e74540c796193db6fe8f9a00815f6c8b95..bcede00802cf0b2bcc3f519d1b94baee32379acc